### ISSUE TITLE
[Bug]: fullScreen as a boolean doesn't appear to work

### ISSUE THREAD

**Issue by 321ckatz123** [tsparticles/tsparticles#4291]
### What happened?

I was in the process of converting from tsParticles 1.x to 2.x and everything worked except being I wasn't on a full screen canvas (it's only displayed on the left side of our app as more of an "accent" design), I noticed that entering text into an input not over the canvas would cause the particles to "reset" (basically as if I completely refreshed the page). I tried to isolate what the issue is and happened to stumble across this comment:

https://github.com/matteobruni/tsparticles/issues/3910#issuecomment-1148754507

I changed fullScreen from `fullScreen: false` to `fullScreen: { enable: false }` and it stopped honoring the keypress in the input (which is what it used to do). Changing it back to the boolean caused the behavior again. Unless I really misunderstand the API (https://particles.js.org/docs/interfaces/Options_Interfaces_IOptions.IOptions.html#fullScreen), that should work the same.

### tsParticles Version

2.1.0

### Which library are you using?

React.js (react-tsparticles, react-particles-js)

### tsParticles Configuration

```json
fullScreen: false,
              particles: {
                number: { value: 40 },
                color: { value: '#688EFF' },
              }
```

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matteobruni** [comment]
https://codepen.io/matteobruni/pen/qBoBVxw you can see it working here, please attach or link a reproducible sample

**Comment by 321ckatz123** [comment]
I could not reproduce even adding all my complexity, BUT it was going through teh React Particles component, not directly calling tsParticles. You wouldn't happen to have a codepen wired up that wires up the React stuff would you, I tried but quickly failed...

**Comment by matteobruni** [comment]
Ok found the issue, it can be seen here: https://codesandbox.io/s/beautiful-dream-yjh5b1?file=/src/App.js

Once set to fullScreen is not unsettling anymore. I will have a look at that after I've fixed all errors that I have in the project right now 😄

**Comment by matteobruni** [comment]
Ok, I found the issue, in React the canvas is not autogenerated, is an existing child of the containing div. Since I don't want to break the existing UI, if the canvas is preexisting I'm trying to restore it to the previous state. That check was failing, it will be released in `2.1.1` (pr #4251)

**Comment by 321ckatz123** [comment]
@matteobruni Yep, that is exactly the behavior I saw. 

I assumed it was a handler incorrectly getting set on fullscren, but React causing the particles to "re-render" in a controlled Textbox also seems quite logical and is what you showed. Thanks for sticking it out with me 👍
